lib/sections/publish.js

```diff
@@ -1,79 +1,62 @@
 var utils = require(__dirname + '/utils.js');
 var uuid  = require('node-uuid');
 var os    = require('os');
-// var i18n  = require('i18n');
-// var time  = require('time');
 
 var publish = function(eventType, args, callback){
   var self = this;
   var queue = self.options.incommigQueue;
-  var klass = self.options.busDriverClassKey;
-
-  payload = [publishMetadata(eventType, args)];
+  var klass = self.options.busClassKey;
+  var payload = [publishMetadata(eventType, args)];
+  payload[0].bus_class_proxy  = '::QueueBus::Driver';
 
   self.queueObject.enqueue(queue, klass, payload, function(err, toRun){
     if(typeof callback === 'function'){ callback(err, toRun); }
   });
-}
+};
 
 var publishAt = function(timestamp, eventType, args, callback){
-  var self = this;
-  var queue = self.options.incommigQueue;
-  var klass = self.options.busPublisherClassKey;
-
-  payload = [publishMetadata(eventType, args)];
-  if(payload["bus_delayed_until"] == null){
-    payload["bus_delayed_until"] = Math.floor(timestamp/1000);
+  var self    = this;
+  var queue   = self.options.incommigQueue;
+  var klass   = self.options.busClassKey;
+  var payload = [publishMetadata(eventType, args)];
+  payload[0].bus_class_proxy  = '::QueueBus::Publisher';
```

Comment:
On a delay publishAt/publishIn, `bus_class_proxy` is always `::QueueBus:: Publisher `